app/Providers/RouteServiceProvider.php

```diff
@@ -18,6 +18,15 @@ class RouteServiceProvider extends ServiceProvider {
      */
     protected $namespace = 'App\Http\Controllers';
 
+    /**
+     * The path to your application's "home" route.
+     *
+     * Typically, users are redirected here after authentication.
+     *
+     * @var string
+     */
+    public const HOME = '/home';
```

Comment:
Shouldn't this be `/dashboard` for us? That's where users should be redirected after authenticating

Reply:
@iccowan Sort of... remember the middleware also handles realops pilots who are not controllers so they do not get routed to /dashboard. From my testing, /home properly routes controllers to the dashboard and logs realops pilots in. I can't explain how this happens... there is no /home route, but it does work. Standing by for your thoughts on this since I do not fully understand the behavior.

Reply:
Hmm okay that doesn't make much sense. I wonder if it's ignoring that route because we're already redirecting them manually since we do all of our authentication through a manual process. In that case, this may not even be necessary (unless we want to leave it to leave Laravel templating).

Reply:
@iccowan Yeah, the reason that I updated it in the first place is that the realops pilot login was failing. Once I updated this everything started working normally again.

Reply:
🤷‍♂️ I'd love to know why that fixed it but if it works, might as well leave it.

Reply:
Confirmed... the `HOME` constant in RouteServiceProvider doesn't do anything. Authenticated users are still routed to the appropriate page regardless of the value.

Reply:
Sounds good - my suggestion would be to remove this completely

Reply:
@iccowan Ok, that's fine - just removed and tested and absolutely nothing changes. There is still one thing here that is functionally a little odd. When a ZTL-rostered controller clicks on the "Login as Realops Pilot" button they get routed to the dashboard. I would prefer that they get routed to the realops page as an authenticated user. I don't see a straightforward way of doing that in LoginController.php - the routine that we have build uses a redirect to /login. Any ideas on how to do this and make it look clean without completely re-engineering this login sequence?

Reply:
I think the issue here was detecting whether the user wants to login as a Realops pilot or as a controller. I think clicking the login button on the Realops page again after being logged in as a controller should work like this?

We may want to add this to the backlog - I would say that's a separate issue from this PR. I did the login sequence quick, and I think it could be made better with some cookies or session tracking to keep track of what is trying to be logged into without guessing.

Reply:
Perhaps a ?back query parameter? That seems to be the standard across many websites, IE redirect to `/login?back=/realops`, and then the login flow parses this out and redirects you to /realops after you’re logged in.